### PR TITLE
Cookbook tutorial improvement.

### LINKED ISSUE #2136: Fixer cookbook missing implementation of abstract methods.
When following https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/COOKBOOK-FIXERS.md step by step, the last point of step 1 can't be completed (`php php-cs-fixer readme > README.rst`) as the following error message is displayed:

```
PHP Fatal error:  Class Symfony\CS\Fixer\Contrib\RemoveCommentsFixer contains 
2 abstract methods and must therefore be declared abstract or implement the 
remaining methods (Symfony\CS\FixerInterface::fix, Symfony\CS\FixerInterface
::getDescription) in PHP-CS-Fixer/Symfony/CS/Fixer/Contrib/RemoveCommentsFixer.php
on line 11
```

I will raise a PR shortly.

### PR DESCRIPTION
**Pull request by g105b**
Fixes #2136 .

### CODE REVIEW

File: COOKBOOK-FIXERS.md [hunk 1]
```diff
@@ -65,8 +65,24 @@ use Symfony\CS\Tokenizer\Tokens;
 /**
  * @author Your name <your@email.com>
  */
-final class RemoveCommentsFixer extends AbstractFixer
+final class RemoveCommentsFixer extends AbstractFixer
```
Comment (keradus) [inline COOKBOOK-FIXERS.md]: please remove that trailing space

**Comment by SpacePossum** [comment]
Line 85 should be removed. Maybe replace the to-do placeholders with some info?

**Comment by SpacePossum** [comment]
@g105b please check see https://github.com/g105b/PHP-CS-Fixer/pull/2

After that this PR needs to be retarged on 1.12 since support on 1.11 ended and we need to get the merge-commit out.

**Comment by keradus** [comment]
I changed the target for you

**Comment by keradus** [comment]
Closing due to #2197

**Pull request closed without merging**